src/ussd_popi_rapidpro.js

```diff
@@ -927,37 +927,195 @@ go.app = function() {
             });
         });
 
-        self.add("state_opt_out", function(name) {
+        self.states.add("state_optout_menu", function(name) {
             return new MenuState(name, {
-                question: $("Do you want to stop getting MomConnect messages?"),
+                question: $("What would you like to do?"),
+                choices: [
+                    new Choice("state_opt_out_reason", $("Stop getting messages")),
+                    new Choice("state_stop_being_part", $("Stop being part of research")),
```

Comment:
state_stop_research might be easier to read than state_stop_being_part beyond this screen.